### ISSUE TITLE
Rasdaman WCS

### ISSUE THREAD

**Issue by rmarzocchi84** [EOX-A/QgsWcsClient2#14]
I try to connect to a Rasdaman WCS but it give me this error: 

```
Your choice:    Rasdaman
URL:                   http://www.gter.it:8080/rasdaman/ows
Server address is valid 
Now testing the connection and response.....
        this may take some time (depending on the server and the volume of its offering)
Response:    An Error occurred: --> Error downloading http://www.gter.it:8080/rasdaman/owsservice=WCS&request=GetCapabilities - server replied: Not Found
 HTTP-Code received: 203
```

It seems to be wrong the request: http://www.gter.it:8080/rasdaman/owsservice=WCS&request=GetCapabilities --> http://www.gter.it:8080/rasdaman/ows?service=WCS&request=GetCapabilities

Perhaps a bug?

**Comment by eox-cs1** [comment]
Hi marzocchi84,
there are 2 issues: 
1) 
You are absolutely right about the url - a correct url should look like this:  
http://www.gter.it:8080/rasdaman/ows?service=WCS&request=GetCapabilities

However, in the QgsWcsClient2  you just need to enter: 
 http://www.gter.it:8080/rasdaman/ows?

2) 
there is a known (and already fixed) bug in rasdaman which requires the following parameter "version=2.0.1"  to be present in a GetCapabilities request. But this is NOT OGC standard.

You can verify this by just entering 
http://www.gter.it:8080/rasdaman/ows?service=WCS&request=GetCapabilities&version=2.0.1 
in any webbrowser. Without the version you will always get an Error message. 

There has already been a rasdaman patch, released in May 2017 (around 24th), which fixes this issue.  Please contact rasdaman for it. 

ciao
Christian

**Comment by rmarzocchi84** [comment]
Thanks for your answer Christian! 

1) sorry.. I confirm: adding the "?" is ok.. the only problem is thay I need to create a new WCS connection. Edit existing connection seems do not work. (I am using qgis 2.18.15)

2) I find the [ticket](http://www.rasdaman.org/ticket/1508) I will contact the developers because it seem to be not fixed in the last version (rasdaman 9.5). They developed a client for QGIS but I think it is not a good idea have three different WCS client in my QGIS   

Many thanks!

**Comment by eox-cs1** [comment]
Hey, 

1) No, you don't need to make a new connection, but you have to also modify the "Name of the Connection" eg. just add a space at the end. 
I am aware that there is still a little glitch with editing WCS connection, sorry about that.  Didn't have time to fix it yet. 

2) Sorry to hear that, I expected that this patch has been already included since it was last May. 
If you need it urgently I could send you a workaround, until you get the patch.

ciao
Christian
